### DISCUSSION TITLE
The docs truly suck

### DISCUSSION THREAD

**Discussion by nicosql** [helix-editor/helix#12014] [General] [answered]
It's just such a chore and I wanted to share that.  Every 3 paragraphs <Link Here> now you're in some unrelated context, examples are beyond generic to the point of uselessness - no indication of what multiple entries should look like.

**Comment by kirawi** [comment, marked as answer]
You are free to submit a PR. Otherwise, this is probably not actionable without understanding what you mean.

**Comment by sicher** [comment]
You might want to change the title to something more constructive.

**Comment by David-Else** [comment]
The docs are extremely minimal, but they **don't suck**.

> Every 3 paragraphs now you're in some unrelated context

Please give an examples.

> examples are beyond generic

Examples are meant to be generic, they show broad concepts. There are not many specific examples as there are so many different ways of doing things and different languages used that showing, for example, something in JavaScript would not be of use to people not using that.

I agree that specific examples would be useful, but that is probably better provided by 3rd party blog and web articles.

> no indication of what multiple entries should look like.

I don't know what this means.

The docs can of course be improved and expanded, but please can you be more specific with your criticism.
